.github/workflows/_docker-build-push.yml

```diff
@@ -48,7 +48,7 @@ jobs:
         id: regex
         with:
           text: ${{ github.ref }}
-          regex: '^(refs\/tags\/${{ inputs.service-name }}\/(v\d+\.\d+\.\d+))|(refs\/heads\/(main))$'
+          regex: '^(refs\/tags\/${{ inputs.service-name }}\/v([^\/]+))|(refs\/heads\/(main))$'
```

Comment:
I suggest not to remote the previous `(v\d+\.\d+\.\d+)` constraint but instead to add an optional `(-[^\/]+)?` suffix to support pre-release versions

Reply:
we can do even more accurate: add full semver regex 🤡